tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -423,8 +423,8 @@ void CheckValidCppVarName(std::string_view var, const std::string &where)
          isValid = false;
 
    if (!isValid) {
-      const auto error =
-         "RDataFrame::" + where + ": cannot define column \"" + std::string(var) + "\". Not a valid C++ variable name.";
+      const auto error = "RDataFrame::" + where + ": cannot define column or variation \"" + std::string(var) +
```

Comment:
I don't understand this. Why specifying about the variation while removing the check on the name validity from Vary at the same time?

Reply:
we still call `CheckValidCppVarName` on the _variation name_, and the error message did not reflect this. I extracted this change to a separate commit to avoid confusion.